src/main/java/org/example/utils/PropertiesLoader.java

```diff
@@ -0,0 +1,34 @@
+package org.example.utils;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.SneakyThrows;
+
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.Charset;
+import java.util.Properties;
+
+public class PropertiesLoader {
+
+    static final String PROPERTIES_FILE = String.format("/%s.properties", System.getProperty("env"));
+    @Getter
+    @Setter
+    private static java.util.Properties propertiesInstance;
```

Comment:
please set propertiesInstance in initializePropertiesInstance() and remove @Getter & @Setter